### PR TITLE
[KOGITO-5005] - Operator print irrelevant metadata in logs

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Many thanks for submitting your Pull Request :heart:! 

Jira issue : https://issues.redhat.com/browse/KOGITO-5005

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster
- [x] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -15,5 +15,5 @@
 - [KOGITO-4894](https://issues.redhat.com/browse/KOGITO-4894) - Remove `image` default value in KogitoRuntime CR (problem in OCP console)
 - [KOGITO-4781](https://issues.redhat.com/browse/KOGITO-4781) - Deployment constantly updates when deployed with custom probes
 - [KOGITO-4993](https://issues.redhat.com/browse/KOGITO-4993) - Operator crash when data-index is created
-
+- [KOGITO-5005](https://issues.redhat.com/browse/KOGITO-5005) - Operator print irrelevant metadata in logs
```
Comment (radtriste) [inline RELEASE_NOTES.md:18]: We still don't have a script which reset the RELEASE NOTES on master once branched ...
Could you remove all the other issues in there ? ...

**Review by radtriste — COMMENTED** [review]
PR check is using kustomize v4. please update it.

**Review by spolti — APPROVED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (vaibhavjainwiz) [reply RELEASE_NOTES.md:5]: done

**Review by radtriste — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]

**Pull request merged**